pkg/webhook/clusterresourceplacementeviction/clusterresourceplacementeviction_validating_webhook_test.go

```diff
@@ -0,0 +1,216 @@
+package clusterresourceplacementeviction
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	admissionv1 "k8s.io/api/admission/v1"
+	authenticationv1 "k8s.io/api/authentication/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	placementv1beta1 "github.com/kubefleet-dev/kubefleet/apis/placement/v1beta1"
+	"github.com/kubefleet-dev/kubefleet/pkg/utils"
+)
+
+func TestHandle(t *testing.T) {
```

Comment:
nit: UTs are being repeated, we can have all the tests consolidated in this file, no need for validator/clusterresourceplacementeviction_test.go